federation/cmd/federation-apiserver/app/server.go

```diff
@@ -196,6 +197,8 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	routes.Version{}.Install(m.Mux, m.HandlerContainer)
```

Comment:
By kubernetes version package, do you mean `pkg/version`? We can move that to `genericapiserver/version` ? (There is no kubernetes specific code there, just some git references)